src/GuiBase/Viewer/Viewer.cpp

```diff
@@ -55,6 +55,24 @@ namespace Ra {
 using namespace Core::Utils; // log
 using namespace glbinding;
 
+#define KMA_VALUE( x ) Gui::KeyMappingManager::KeyMappingAction Gui::Viewer::x;
+KeyMappingViewer;
+#undef KMA_VALUE
+
+void Gui::Viewer::registerKeyMapping() {
+    m_keyMappingContext = Gui::KeyMappingManager::getInstance()->getContext( "ViewerContext" );
+    if ( m_keyMappingContext.isInvalid() )
+    {
+        LOG( logINFO )
+            << "ViewerContext not defined (maybe the configuration file do not contains it";
+    }
+
+#define KMA_VALUE( XX ) \
+    XX = Gui::KeyMappingManager::getInstance()->getActionIndex( m_keyMappingContext, #XX );
+    KeyMappingViewer
+#undef KMA_VALUE
```

Comment:
In case the context is invalid we should return and not try to access it, otherwise this will crash